Structures/Structures/Consigment.cs

```diff
@@ -0,0 +1,75 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace Structures
+{
+    public struct Consigment
+    {
+        int quantity;
+        double price;
+
+        public int Quantity 
+        { 
+            get => quantity;
+            set
+            {
+                if (value < 0) throw new ArgumentException("Количество товара должно быть положительным");
+                quantity = value;
+            }
+        }
+
+        public double Price
+        {
+            get => price;
+            set
+            {
+                if (value < 0) throw new ArgumentException("Цена должна быть положительной");
+                price = Math.Round(value, 2);
+            }
+        }
+
+        public double Cost
+        {
+            get => Math.Round(quantity * price, 2);
+        }
```

Comment:
Можно меньше инфраструктурного кода
```suggestion
        public double Cost => Math.Round(quantity * price, 2);
        
```